sample/sample8-middlewares/UserController.ts

```diff
@@ -0,0 +1,56 @@
+import {Request} from "express";
+import {Controller} from "../../src/decorator/controllers";
+import {Get, Post, Put, Patch, Delete, Use} from "../../src/decorator/methods";
+import {Req, Next} from "../../src/decorator/params";
+
+@Controller()
+export class UserController {
+
+
+    @Use()
+    middleware( @Next() next: any) {
+      console.log("calling middleware");
+      next();
+    }
+
+    @Get("/users", { jsonResponse: true })
+    getAll() {
+        return [
+            { id: 1, name: "First user!" },
+            { id: 2, name: "Second user!" }
+        ];
+    }
+
+    @Get("/users/:id")
+    getOneMdlw( @Req() request: Request, @Next() next: any) {
```

Comment:
I haven't used next function a lot, and I can't get one point... Do we really need to use @Next here? I makes sense for me if we used it in something that is running before this controller action (`getOneMdlw`), but this controller action getOneMdlw is the last in the stack, and there is nothing to call after it using next. Or am I wrong?